WebAuthn docs update (#25922)
mdn/content: files/en-us/web/api/publickeycredential/authenticatorattachment/index.md

---
title: "PublicKeyCredential: authenticatorAttachment property"
short-title: authenticatorAttachment
slug: Web/API/PublicKeyCredential/authenticatorAttachment
page-type: web-api-instance-property
browser-compat: api.PublicKeyCredential.authenticatorAttachment
---

{{APIRef("Web Authentication API")}}{{securecontext_header}}

The **`authenticatorAttachment`** read-only property of the

> {{domxref("PublicKeyCredential")}} interface is a string that indicates the mechanism by which the WebAuthn implementation is attached to the authenticator at the time the associated {{domxref("CredentialsContainer.create()","navigator.credentials.create()")}} or {{domxref("CredentialsContainer.get()","navigator.credentials.get()")}} call completes.

Reviewer: Something feels a bit off to me with the wording here, but I'm getting the feeling it's because "of course you can't use 'attachment' to define `authenticatorAttachment`" and it's not leaving a lot of good words available for use for this seemingly straight-forward definition 😅 

Maybe something like this instead?
```suggestion
{{domxref("PublicKeyCredential")}} interface is a string that indicates the general category of authenticator used during the associated {{domxref("CredentialsContainer.create()","navigator.credentials.create()")}} or {{domxref("CredentialsContainer.get()","navigator.credentials.get()")}} call.
```

Author: Updated! That is so much better; the original was horribly wordy and awkward. I wasn't sure how to simplify without affecting the message...
